openmc/deplete/results_list.py

```diff
@@ -361,6 +361,7 @@ def export_to_materials(self, burnup_index, nuc_with_data=None) -> Materials:
             mat_id = str(mat.id)
             if mat_id in result.mat_to_ind:
                 mat.volume = result.volume[mat_id]
+                mat.set_density('sum')
```

Comment:
How did you catch this bug? It seems like it would result in only tiny differences compared to the expected simulation results. Amazing catch.

Reply:
Well, I was running some silly case and put a power several orders of magnitude too high, which depleted the hell of the material... and then I was wondering why when I restarted the depletion (after using `ResultsList.export_to_materials`), the densities suddenly changed. After some digging it led me to this. But yes, this would be a very small error in the normal course of things :smile: